lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php

```diff
@@ -614,7 +614,7 @@ private function completeIdGeneratorMapping(ClassMetadataInfo $class): void
 
                 $sequenceGenerator = new SequenceGenerator(
                     $this->em->getConfiguration()->getQuoteStrategy()->getSequenceName($definition, $class, $this->getTargetPlatform()),
-                    $definition['allocationSize']
+                    (int) $definition['allocationSize']
```

Comment:
Since the `+` operator is used here, this must be fine:

https://github.com/doctrine/orm/blob/95408cd8e41867c38ac9cc2395404e391b3df389/lib/Doctrine/ORM/Id/SequenceGenerator.php#L78